docs/build_a_connector.rst

```diff
@@ -16,11 +16,19 @@ The first thing you’ll need to do is create a new folder for your connector.
 
 Inside the folder, create two files.  The first should be named __init__.py and should be empty.  The second will have the same name as your folder - this is the file which will have your connector’s code.  For example, in the airtable folder this file is called airtable.py and in the hustle folder it’s called hustle.py.
 
-The directory should look like this::
+The directory should look like this:
 
+.. code-block:: python
     yourconnectorname/
     __init__.py
         yourconnectorname.py
+		
+Next, add the reference to your connector to parsons/__init__.py.  Specifically, open parsons/__init__.py, scroll to the end of the other imports, and add the following:
```

Comment:
Can you put the references to parsons/__init__.py in backticks `` so they show up as code?  Github works the same way, fyi, here's `parsons/__init__.py` using backticks.  (Except Github uses single backticks, and our docs use double.)